### DISCUSSION TITLE
开发者例会议题收集（20221101）

### DISCUSSION THREAD

**Discussion by chuntaojun** [polarismesh/polaris#779] [Meeting]
# 开发者例会议题收集（20221018）

## 会议信息

andrewshan(单家骏) 邀请您参加腾讯会议
会议主题：Polaris社区开发者双周会
会议时间：2022/09/20 19:30-20:00 (GMT+08:00) 中国标准时间 - 北京
重复周期：2022/07/12-2022/10/04 19:30-20:00, 每两周 (周二)

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/dm/kBemx9IzxmGE

#腾讯会议：701-1815-5065

复制该信息，打开手机腾讯会议即可参与

**Comment by chuntaojun** [comment]
议题列表

1. 社区重新讨论北极星是否需要推空保护这一措施
   - 移除原本SDK中的推空保护逻辑（polaris-java、polaris-go） 
3. 原生的spring cloud 如何接入北极星的注册中心配置中心并实现服务治理？
4. tars协议怎么用sidecar模式? 也就是说自定义协议如何在sidecar拆包 做负载均衡呢？
5. 社区同学 @lhiamgeek 分享 polaris-controller 的原理、开发体会
6. @andrewshan 分享 polaris-java-agent 的设计

**Reply by andrewshan** [reply]
https://github.com/polarismesh/polaris/issues/353

**Comment by gtyi999** [comment]
3 原生的spring cloud 如何接入北极星的注册中心配置中心并实现服务治理？

**Reply by dongyinuo** [reply]
https://github.com/Tencent/spring-cloud-tencent 看一下 这个是否能够帮助到您

**Reply by reallovelei** [reply]
4. tars协议怎么用sidecar模式? 也就是说自定义协议如何在sidecar拆包 做负载均衡呢？

**Reply by chuntaojun** [reply]
> gtyi999

目前支持 spring cloud tencent，即通过 pom 引入依赖的方式以及使用 java-agent

**Comment by lhiamgeek** [comment]
polaris controller 原理及定时对账实现ppt
[polaris.pptx](https://github.com/polarismesh/polaris/files/9909799/polaris.pptx)

**Comment by chuntaojun** [comment]
主题: Polaris社区开发者双周会
日期: 2022-11-01 19:25:30
录制文件：https://meeting.tencent.com/v2/cloud-record/share?id=f6a785c4-912c-4b1a-acfd-2e6f5fb575cd&from=3
访问密码：wDeV
